pkg/controller/plan/kubevirt.go

```diff
@@ -1942,6 +1961,12 @@ func (r *KubeVirt) secretDataSetterForCDI(vmRef ref.Ref) func(*core.Secret) erro
 	}
 }
 
+func (r *KubeVirt) isSecretExist(name, namespace string) (err error) {
```

Comment:
this function should either return a boolean that indicates whether the secret exists or renamed